renv.lock

```diff
@@ -697,15 +697,15 @@
     },
     "projectlive.modules": {
       "Package": "projectlive.modules",
-      "Version": "0.4.1",
+      "Version": "0.5",
```

Comment:
The module is working well and the appropriate plots are there but there is some room to improve the aesthetics of the plot. Some suggestions are below:
1. Font of text needs to be smaller for the top text and the plot headers
2. Suggest using `infoboxes` to form headers of the plots
3. The DT and slider are squished together to the left side .. need to add spacing to make them horizontally aligned and evenly spaced
4. Suggest adding user info text associated with each plot (i.e. line that denotes time window) as its own entity and not as plot title

Reply:
some examples here: https://jkunst.com/blog/posts/2020-06-26-valuebox-and-sparklines/